Cookbook/Current.md

```diff
@@ -0,0 +1,22 @@
+# How to use `Current` in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global implementation of the `World` file which is located in the `BabylonDependencies.framework`.
+
+### Properties with default values
```

Comment:
What is the value of documenting these here?

Reply:
you will see when you need to create one here.

Reply:
it's a guide